cou/steps/hypervisor.py

```diff
@@ -126,9 +126,8 @@ def machines(self) -> list[COUMachine]:
         """
         return self._machines
 
-    @property
-    def azs(self) -> AZs:
-        """Returns a list of AZs defined in individual applications.
+    def azs(self, target: OpenStackRelease) -> AZs:
```

Comment:
Without being property, I will suggest renaming it to `get_azs`. What do you say?